tests/e2e/apps_test.go

```diff
@@ -125,7 +126,10 @@ var _ = Describe("Apps", func() {
 			appName := generateGUID("app")
 			appGUID = createBuildpackApp(space1GUID, appName)
 			pkgGUID = createBitsPackage(appGUID)
+			uploadTestApp(pkgGUID, defaultAppBitsFile)
 			buildGUID = createBuild(pkgGUID)
+			waitForDroplet(buildGUID)
```

Comment:
I guess you added those to fix the error that you get when the droplet is not set. However, you should not do that. Deleting an app with no droplet should work, i.e. the `delete an app` e2e test should not change